packages/assets/scss/03-organisms/_page-banner.scss

```
@@ -22,10 +22,10 @@ $page-banner-height: 330px;
   .decision-tree & {
 
     h1 {
-      font-size: $fonts-2xlarge;
+      font-size: $fonts-xlarge;

```

```suggestion
      font-size: $fonts-xlarge;
```
is there a reason that the h1 in decision tree is changed?

---

I think I made a mistake here. Thanks @clairesunstudio . I will delete that.